qemu/tests/plug_cdrom.py

```diff
@@ -0,0 +1,115 @@
+import logging
+import ast
+import re
+
+from virttest import error_context
+from virttest import utils_test
+
+from provider.plug_block_devices import PlugManager
```

Comment:
Tried test with patch https://github.com/autotest/tp-qemu/pull/1735, hit ERROR: No module named 'provider.plug_block_devices', the module should be updated~